modules/microshift-config-yaml.adoc

```diff
@@ -27,20 +27,32 @@ node:
   hostnameOverride: "" <5>
   nodeIP: "" <6>
 apiServer:
-  subjectAltNames: [] <7>
+  advertiseAddress:
+    - 10.44.0.0/32 <7>
+  subjectAltNames: [] <8>
 debugging:
-  logLevel: "Normal" <8>
+  logLevel: "Normal" <9>
 ----
 <1> Base domain of the cluster. All managed DNS records will be subdomains of this base.
 <2> A block of IP addresses from which Pod IP addresses are allocated.
 <3> A block of virtual IP addresses for Kubernetes services.
 <4> The port range allowed for Kubernetes services of type NodePort.
 <5> The name of the node. The default value is the hostname.
 <6> The IP address of the node. The default value is the IP address of the default route.
-<7> Subject Alternative Names for API server certificates.
-<8> Log verbosity. Valid values for this field are `Normal`, `Debug`, `Trace`, or `TraceAll`.
+<7> Specifies the IP address from which the API server is advertised to members of the cluster. The default value is `10.44.0.0/32`.
+<8> Subject Alternative Names for API server certificates.
+<9> Log verbosity. Valid values for this field are `Normal`, `Debug`, `Trace`, or `TraceAll`.
 
 [IMPORTANT]
 ====
 Restart {product-title} after changing any configuration settings to have them take effect. {product-title} reads the configuration file only on start.
 ====
+
+[id="microshift-yaml-custom_{context}"]
+== Custom settings
+You must change any settings that should override the defaults before starting {product-title}.
+
+advertiseAddress::
+The `apiserver.advertiseAddress` flag specifies the IP address on which to advertise the API server to members of the cluster. This address must be reachable by the cluster. You can set a custom IP address here, but you must also add the IP address to a host interface. Customizing this parameter preempts {product-title} from adding the an IP address to the `br-ex` network interface.
++
+If unset, the default value is `10.44.0.0/32`. It will be set to the next immediate subnet after the service network. For example, when the service network is `10.43.0.0/16`, the `advertiseAddress` is set to `10.44.0.0/32`.
```

Comment:
The value is calculated based on the service network CIDR. So it's `10.44.0.0/32` only when the service network is `10.43.0.0/16`.

Reply:
@pliurh [Here](https://github.com/openshift/openshift-docs/pull/63275), we removed CIDR. Is it OK to mention in the current PR?

Reply:
We can use `serviceNetwork` instead.